M2/Macaulay2/m2/help.m2

```diff
@@ -453,13 +453,15 @@ setAttribute(viewHelp#0, ReverseDictionary, symbol viewHelp)
 
 infoHelp = method(Dispatch => Thing)
 infoHelp Thing := key -> (
-    if key === () then infoHelp "Macaulay2Doc";
+    if key === () then return infoHelp "Macaulay2Doc";
     tag := infoTagConvert makeDocumentTag(key, Package => null);
     if getenv "INSIDE_EMACS" == "" then chkrun ("info " | format tag)
     -- used by M2-info-help in M2.el
-    else print("-* infoHelp: " | tag | " *-");)
+    else print("-*" | " infoHelp: " | tag | " *-");)
```

Comment:
I was staring at this line for 2 minutes trying to figure out what's happening lol